tortoise/tortoise.go

```diff
@@ -1116,7 +1129,10 @@ func (t *turtle) ballotFilterForHealing(candidateLayerID types.LayerID, logger l
 			logger.With().Error("inconsistent state: ballot not found", ballotID)
 			return false
 		}
-		return lid.Uint32() > t.BadBeaconVoteDelayLayers && lid.Sub(t.BadBeaconVoteDelayLayers).After(candidateLayerID)
+		// if candidate layer is 11 and vote with bad beacon is in 12, but current is 100
+		// we should obviously count such vote.
+		// is there any case
```

Comment:
maybe this is a misinterpretation on my part? by delaying votes for beacon for N layers, do you think Tal means `current-ballotLayer`?

Reply:
my interpretation is that we shouldn't count votes with bad beacon in current epoch

> the tortoise does count the ballots from previous epochs regardless of the beacon, but because there was a partition, each side sees less total weight and thus generates double the number of ballots

Reply:
that's not my understanding. Tal explicitly said delay by X layers. there was no mention about current epoch. maybe this is worth clarifying with him.

Reply:
> Tal explicitly said delay by X layers

i would interpret that as delay by x layers from current layer, so that it can be less than single epoch. but i asked in the channel